### ISSUE TITLE
Implement translations

### ISSUE THREAD

**Issue by 0x1AE7F** [0x1AE7F/Protea#3]
Translations are be placed in `src/lib/langs/<localeName>/<pageName>`, for example:

`src/lib/langs/en_US/login.json` for the login component.

Contents of the json files should look similar to the following example:
```
{
	"login_prompt": "Please login to continue",
	"label_username_email": "Username or E-Mail",
	"label_password": "Password",
	"btn_login": "Login",
	"btn_login_loading": "Please wait"
}
```
These will then be referenced in the components.

We need to find a way to store the users language preference ( database? ) and then automatically get the right translations

**Comment by SneakyNieky37** [comment]
perhaps have it stored in the database and check it whenever the user logs in. the login page should theb always be in English

**Comment by 0x1AE7F** [comment]
Yeah, we could pass the preferred lang when creating the users profile. We could also use the .env file to specify a "fallback" language incase the user is not logged in.

**Comment by 0x1AE7F** [comment]
Yeah, we could pass the preferred lang when creating the users profile. We could also use the .env file to specify a "fallback" language incase the user is not logged in.

Ill work on the database stuff after Ive got some sleep

**Comment by 0x1AE7F** [comment]
Implementation in PR #5
